docs/index.html

```diff
@@ -2,191 +2,615 @@
+    submodules:
+    - icon-button
+  inline-notice:
+    ds-name: alert-notice
+    ds-version: 2.2
+    submodules:
+    - icon
+  lightbox-dialog:
+    ds-components:
+    - ds-name: modal
+      ds-version: 2.1
+    - ds-name: bottom-sheet
+      ds-version: 2.1
+    submodules:
+    - button
+    - icon-button
+  less:
+    ds-version: 1.0
+  link:
+    ds-name: text-link
+    ds-version: 2.0
+  listbox:
+    ds-name: dropdown
+    ds-version: 2.2
+    submodules:
+    - icon
+  listbox-button:
+    ds-name: dropdown
+    ds-version: 2.2
+    submodules:
+    - button
+    - icon
+  menu:
+    ds-name: dropdown
+    ds-version: 2.3
+    submodules:
+    - icon
+  menu-button:
+    ds-name: dropdown
+    ds-version: 2.3
+    submodules:
+    - button
+    - icon
+    - menu
+  page-grid:
+    ds-version: 1.0.0
+  page-notice:
+    ds-name: alert-notice
+    ds-version: 3.0
+    submodules:
+    - icon
+  pagination:
+    ds-name: pagination
+    ds-version: 2.0
+    submodules:
+    - icon-link
+  panel-dialog:
+    ds-name: panel
+    ds-version: 2.0
+    submodules:
+    - button
+    - icon-button
+  progress-bar:
+    ds-name: progress-bar
+    ds-version: 3.1
+  progress-spinner:
+    ds-name: progress-spinner
+    ds-version: 3.1
+    submodules:
+    - icon
+  progress-stepper:
+    ds-name: progress-stepper
+    ds-version: 4.0
+    submodules:
+    - icon
+  radio:
+    ds-name: radio-button
+    ds-version: 2.0
+  section-notice:
+    ds-name: alert-notice
+    ds-version: 3.0
+    submodules:
+    - icon
+  section-title:
+    ds-version: 4.0
+  segmented-buttons:
+    ds-name: segmented-button
+    ds-version: 1.0
+  select:
+    ds-name: dropdown
+    ds-version: 2.2
+    submodules:
+    - icon
+  signal:
+    ds-name: signal
+    ds-version: 1.0
+  skeleton:
+    ds-name: skeleton
+    status: beta
+    ds-version: 1.0
+  snackbar-dialog:
+    ds-name: snackbar
+    ds-version: 2.0
+    submodules:
+    - button
+  split-button:
+    ds-group: button
+    ds-name: split-button
+    ds-version: 1.6
+    submodules:
+    - button
+    - menu-button
+  star-rating:
+    ds-name: review-rating
+    ds-version: 1.0
+  star-rating-select:
+    ds-name: review-input
+    ds-version: 1.0
+  switch:
+    ds-name: switch
+    ds-version: 2.1
+  tabs:
+    ds-name: tabs
+    ds-version: 2.1
+  textbox:
+    ds-name: form-field
+    ds-version: 2.2
+    submodules:
+    - icon-button
+  toast-dialog:
+    ds-version: 2.2
+    submodules:
+    - button
+    - icon-button
+  toggle-button:
+    ds-name: toggle-button
+    ds-version: 1.0
+    submodules:
+    - icon
+  toggle-button-group:
+    ds-name: toggle-button-group
+    ds-version: 1.0
+    submodules:
+    - toggle-button
+  tooltip:
+    ds-name: tooltip
+    ds-version: 2.0
+    submodules:
+    - icon-button
+  tourtip:
+    ds-name: tourtip
+    ds-version: 2.0
+    submodules:
+    - icon-button
+  typography:
+    ds-version: 1.1
+# Schema Proposal 2
+module_metadata:
```

Comment:
Schema 2: Module has optional submodules field, ds-component field (containing all the DS-relevant metadata), and status field